sdk/storage/azure-storage-blob/swagger/README.md

```diff
@@ -11,12 +11,12 @@
 ```ps
 cd <swagger-folder>
 mvn install
-autorest --java --use:@autorest/java@4.0.x
+autorest --java --use:@autorest/java@4.1.x
 ```
 
 ### Code generation settings
 ``` yaml
-input-file: https://raw.githubusercontent.com/seanmcc-msft/azure-rest-api-specs/95d8b1a4c972eafa176dce585c45e7e0611fb04e/specification/storage/data-plane/Microsoft.BlobStorage/preview/2021-04-10/blob.json
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/e515b6251fdc21015282d2e84b85beec7c091763/specification/storage/data-plane/Microsoft.BlobStorage/preview/2021-04-10/blob.json
```

Comment:
This is taken from https://github.com/Azure/azure-sdk-for-net/blob/d7563fd813d21492fe210e4902855c0ed60c7d38/sdk/storage/Azure.Storage.Blobs/src/autorest.md